kinds/5951.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Relay Access Request
+description: Request write-access to private relays
+---
+
+Clients can use this job to request access to private relays, either by mentioning a specific relay's NIP-11 pubkey, or by creating an empty request and relying on DVMs to suggest relays to the user.
+
+# Input
+
+The URL (eg `wss://relay.mostr.pub/`), if any.
+If no input is provided, the client is relying on DVMs to suggest relays to the user.
```

Comment:
What's the use case for this?

Reply:
You might want to join a private relay but you don't know which one. DVMs will respond and say "You can join wss://nostr.wine/, just pay us X sats"